app/views/jobs/tango_status.html.erb

```diff
@@ -0,0 +1,277 @@
+<% @title = "Tango Status" %>
```

Comment:
Could we clean up the tabulation in this file?  2 spaces everywhere?  This looks like a weird mix of different space amounts and hard tabs.